docs/1_installation.md

```diff
@@ -9,6 +9,6 @@ In order to use ReVanced on your Android device, ReVanced Manager must be instal
 
 ## ⏭️ What's next
 
-The next page will guide you through patching an application.
+The next page will guide you through patching an application and using ReVanced Manager.
 
-Continue: [🪛 Usage](2_usage.md)
+Continue: [📱 Usage](2_usage.md)
```

Comment:
If you update icons, you have to update them everywhere, such as in the docs readme or table of context.

Reply:
Okay, I forgot to edit all icons. I only changed it because the in most of the pages the Usage page icon was the double hammer, which was already in use for Building page. And the screwdriver didnt seem fitting